esmvalcore/config-user.yml

```diff
@@ -85,17 +85,20 @@ profile_diagnostic: false
 #  CORDEX: /mnt/lustre02/work/ik1017/C3SCORDEX/data/c3s-cordex/output
 #  OBS: /mnt/lustre02/work/bd0854/DATA/ESMValTool2/OBS
 #  OBS6: /mnt/lustre02/work/bd0854/DATA/ESMValTool2/OBS
-#  obs4mips: /mnt/lustre02/work/bd0854/DATA/ESMValTool2/OBS 
-#  ana4mips: /mnt/lustre02/work/bd0854/DATA/ESMValTool2/OBS 
+#  obs4mips: /mnt/lustre02/work/bd0854/DATA/ESMValTool2/OBS
+#  ana4mips: /mnt/lustre02/work/bd0854/DATA/ESMValTool2/OBS
+#  native6: /mnt/lustre02/work/bd0854/DATA/ESMValTool2/RAWOBS
+#  RAWOBS: /mnt/lustre02/work/bd0854/DATA/ESMValTool2/RAWOBS
 #drs:
 #  CMIP6: DKRZ
 #  CMIP5: DKRZ
 #  CMIP3: DKRZ
-#  CORDEX: DKRZ
-#  OBS: DKRZ
-#  OBS6: DKRZ
-#  obs4mips: DKRZ
-#  ana4mips: DKRZ
+#  CORDEX: BADC
```

Comment:
Shouldn't it be: `# CORDEX: default`?

Reply:
Here is an example filename:
```
EUR-11/KNMI/MPI-M-MPI-ESM-LR/historical/r1i1p1/KNMI-RACMO22E/v1/mon/tas/v20190619/tas_EUR-11_MPI-M-MPI-ESM-LR_historical_r1i1p1_KNMI-RACMO22E_v1_mon_195001-195012.nc
```
so the directories are named according to
https://github.com/ESMValGroup/ESMValCore/blob/5c094eecac2f2ff652bb34bbf883c8c5ea4ad419/esmvalcore/config-developer.yml#L233
while `default` is a structure with no subdirectories:
https://github.com/ESMValGroup/ESMValCore/blob/5c094eecac2f2ff652bb34bbf883c8c5ea4ad419/esmvalcore/config-developer.yml#L231

Reply:
I see. Thanks for pointing this out